prime-router/src/main/kotlin/transport/RedoxTransport.kt

```diff
@@ -89,9 +98,19 @@ class RedoxTransport() : ITransport {
             .responseJson()
         // TODO: store the result id when we get line level tracking
         return when (result) {
-            is Result.Success -> true
-            is Result.Failure -> false
+            is Result.Success -> {
+                logger.info("Redox call succeeded: $itemId")
+                true
```

Comment:
Again, I messed in this code as well, so you'll see conflicts.   I think you should definitely log the URL its sending to; config problems will be much easier to figure out.